scripts/build/Jenkins/Jenkinsfile

```diff
@@ -845,8 +845,10 @@ try {
                     // Stash the INCREMENTAL_BUILD_SCRIPT_PATH and EBS_SNAPSHOT_SCRIPT_PATH since all nodes will use it
                     stash name: 'incremental_build_script',
                           includes: INCREMENTAL_BUILD_SCRIPT_PATH
-                    stash name: 'ebs_snapshot_script',
+                    if (fileExists(EBS_SNAPSHOT_SCRIPT_PATH)) {
```

Comment:
Since the script is only used for the snapshot pipeline, we can also only stash if the CREATE_SNAPSHOT env var is set to avoid stashing for PR builds for example.

Reply:
Reviewed with Mike, GetBuildEnvVars() is called after the stash so it won't be available at this point in the pipe.

Reply:
The build configuration is loaded but not directly expressed as envvars until after the stash occurs. I can see if we can work around that in a separate PR.